Tests/Admin/AdminTest.php

```diff
@@ -1524,18 +1524,6 @@ public function testGetSubject()
         $admin->setModelManager($modelManager);
 
         $admin->setRequest(new Request(array('id' => 'azerty')));
-        $this->assertFalse($admin->getSubject());
```

Comment:
These tests were written to verify compliance with regular expression `^[0-9A-Fa-f\-]+$`.
These tests are now nothing to test.